File: Baekjoon/Python/BOJ_PS/Math/Q5355-MarsMath.py
# 화성 수학 / 수학
for i in range(int(input())):
    expressions = list(map(str, input().split()))
    num = float(expressions[0])

    for j in range(1, len(expressions)):
        if expressions[j] == '@':
            num *= 3
        elif expressions[j] == '%':
            num += 5
        elif expressions[j] == '#':
            num -= 7

    print(format(num, ".2f"))

File: Baekjoon/Swift/BOJ_PS/BOJ_PS/Math/Q5355-MarsMath.swift
//
//  Q5355-MarsMath.swift
//  BOJ_PS
//
//  Created by thoonk on 2022/07/30.
//  화성 수학 / 수학

import Foundation

for _ in 0..<Int(readLine()!)! {
    let expressions = readLine()!.split(separator: " ").map { String($0) }
    
    var num = Double(expressions[0])!
    
    for i in 1..<expressions.count {
        if expressions[i] == "@" {
            num *= 3
        } else if expressions[i] == "%" {
            num += 5
        } else if expressions[i] == "#" {
            num -= 7
        }
    }
    
    print(String(format: "%.2f", num))
}

